automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebookPyKernelSpec.scala

```diff
@@ -63,6 +63,17 @@ class NotebookPyKernelSpec extends ClusterFixtureSpec {
       }
     }
 
+    "should update dateAccessed if the notebook is open" in { clusterFixture =>
+      withWebDriver { implicit driver =>
+        withNewNotebook(clusterFixture.cluster) { notebookPage =>
+          val firstApiCall = Leonardo.cluster.get(clusterFixture.billingProject, clusterFixture.cluster.clusterName)
+          //Sleeping for 90s to simulate idle notebook
```

Comment:
Comment is fine but I was actually suggesting a log msg to make the test more debuggable.